README.md

```diff
@@ -124,4 +124,6 @@ And finally we can see the video generated by Crypress for this integration test
 https://user-images.githubusercontent.com/16966767/155645494-7d741f23-850b-42be-b881-8599133e9d18.mp4
 
 
+Other e2e testing
 
+video-none :(
```

Comment:
please jus need to put the video into github directory directly, do you have any problem?